lib/DTLZ/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.0.1"
```

Comment:
I suggest we stick to the `version` attribute of the `Benchmark` class defined in `benchmark.py`.

Reply:
It is also there, (name in the class is ``version``, not ``__version__``.  Should I just delete this occurrence?

Reply:
I think a link is missing in the "there". for benchmarks let's keep it as a class attribute `version`. For Python packages (e.g., deephyper, deephyper_benchmarks) it should be `__version__` in the root `__init__.py` of the package.